src/SUMMARY.md

```diff
@@ -37,6 +37,7 @@
       - [Daily Active Users (DAU and MAU)](cookbooks/dau.md)
       - [Active DAU (aDAU)](cookbooks/active_dau.md)
       - [Retention](cookbooks/retention.md)
+    - [Additional Properties](cookbooks/additional_props.md)
```

Comment:
Could you put this under "Accessing and working with BigQuery"? (last item)

```suggestion
    - [Accessing Additional Properties](cookbooks/additional_props.md)
```

(also, it looks capitalization here is a mess, but that's not your fault and we can leave it for later) 😄